DopplerJobsServer/Program.cs

```diff
@@ -16,6 +17,28 @@ public static void Main(string[] args)
         public static IHostBuilder CreateHostBuilder(string[] args) =>
             Host.CreateDefaultBuilder(args)
                 .UseSerilog((hostContext, loggerConfiguration) => loggerConfiguration.ReadFrom.Configuration(hostContext.Configuration))
+                .ConfigureAppConfiguration((hostContext, configurationBuilder) =>
+                {
+                    // It is if you want to override the configuration in your
+                    // local environment, `*.Secret.*` files will not be
+                    // included in git.
+                    configurationBuilder.AddJsonFile("appsettings.Secret.json", true);
+
+                    // It is to override configuration using Docker's services.
+                    // Probably this will be the way of overriding the
+                    // configuration in our Swarm stack.
+                    configurationBuilder.AddJsonFile("/run/secrets/appsettings.Secret.json", true);
+
+                    // Configuration for Docker Windows containers.
+                    configurationBuilder.AddJsonFile("/ProgramData/docker/secrets/appsettings.Secret.json", true);
```

Comment:
What do you think about using the complete path in a Windows format?

```suggestion
                    configurationBuilder.AddJsonFile("C:\\ProgramData\\Docker\\secrets\\appsettings.Secret.json", true);
```

Reply:
Yes I think that is correct this change I'll do